fabric/patches/2_0001-Extension-point-for-go-build-tag-enabled-additional-.patch

```diff
@@ -0,0 +1,79 @@
+From 7abcf2a461eb6f6289d797d0142de4f7e1fdb5ff Mon Sep 17 00:00:00 2001
+From: michael steiner <michael.steiner@intel.com>
+Date: Fri, 31 Jul 2020 14:21:19 -0700
+Subject: [PATCH 1/2] Extension point for go build-tag enabled additional SCCs
+
+Signed-off-by: michael steiner <michael.steiner@intel.com>
+---
+ internal/peer/node/sccs_opt.go | 31 +++++++++++++++++++++++++++++++
+ internal/peer/node/start.go    |  9 ++++++++-
+ 2 files changed, 39 insertions(+), 1 deletion(-)
+ create mode 100644 internal/peer/node/sccs_opt.go
+
+diff --git a/internal/peer/node/sccs_opt.go b/internal/peer/node/sccs_opt.go
+new file mode 100644
+index 000000000..d46fa4811
+--- /dev/null
++++ b/internal/peer/node/sccs_opt.go
```

Comment:
looking in this directory `sccs_opt.go` looks a bit at the wrong place. Works fine for now, but for upstream maybe better in `github.com/hyperledger/fabric/core/scc`?

Reply:
yep, `internal` is not the best case, i would think though better to add as subdir inside scc, not directly in top-level?

Reply:
we will figure out what the proper place is by creating this PR on the Fabric repo :) maintainers will tell :D